content/en/release-notes/_index.md

```diff
@@ -12,6 +12,53 @@ version: 'releases'
 title: Releases
 ---
 
+# Apache Log4j announcement
+
+Following news of [Apache Log4j 2's vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), new patches for all supported versions of Corda Open Source, Corda Enterprise, and CENM are in progress.
+
+Your individual CorDapps do not need to be patched as they inherit Apache Log4j from the Corda runtime.
+
+Check the list below for expected patch release dates for your version of Corda or CENM. Use the [mitigation guide](#what-you-can-do-now) to reduce your risk before upgrading to the new patch.
+
+### Apache Log4j patch release timetable
+
+| Version with new patch | Patch target shipping date    | Mitigation available |
+| :------------- | :------------- | :------------- |
+| Corda OS/CE 4.8.4      | Dec 13      | Yes |
+| Corda OS/CE 4.6.6      | Dec 14      | Yes |
+| Corda OS/CE 4.5.7      | Dec 15      | Yes |
+| Corda OS/CE 4.7.4      | Dec 15      | Yes |
+| Corda OS/CE 4.4.9      | Dec 16      | Yes |
+| Corda OS/CE 4.3.9      | Dec 16      | Yes |
+| CENM 1.2.4      | Dec 14      | No |
+| CENM 1.5.2      | Dec 15      | Yes |
+| CENM 1.3.3      | Dec 16      | Yes |
+| CENM 1.4.2      | Dec 17      | Yes |
+
+{{< note >}}
+These patch releases are valid for the stated supported versions of Corda and CENM only. If you are not using a supported version of Corda or CENM, please upgrade to one of the above versions.
+{{< /note >}}
+
+## What you can do now
+
+While you wait for the release of the required emergency patch, you can apply one of the following steps to mitigate the threat implied by the log4j Zero-day vulnerability, as referenced in https://nvd.nist.gov/vuln/detail/CVE-2021-44228.
+
+### For Corda 4.3.8 and above, and CENM 1.3 and above
```

Comment:
Would be more correct to say Corda 4.3 and above as the original GA release is eligible for the tactical fix